src/scripts/bazel_build.sh

```diff
@@ -18,10 +18,26 @@ cd $CPP_WORKSPACE_DIR \
 && mv bazel-bin/query $BIN_DIR
```

Comment:
Instructions to build `hyperon_das_atomdb_cpp` are missing in the first block of this file, as follows:

```suggestion
&& mv bazel-bin/query $BIN_DIR \
&& $BAZELISK_BUILD_CMD \
      //hyperon_das_atomdb_cpp:hyperon_das_atomdb_cpp_wheel \
      --define=ATOMDB_VERSION=0.8.11 \
&& mv bazel-bin/hyperon_das_atomdb_cpp/*.whl "$BIN_DIR"
```